### DISCUSSION TITLE
是否可以增加时间同步功能？

### DISCUSSION THREAD

**Discussion by cheryhx** [volatile-static/Chartero#83] [Ideas]
zotero最激励我看文献的插件，没有之一！！非常好用，感谢大大～！
有时甚至通过阅读时间判断对研究的重要性
但是经常三端（win, os, ios)看文献，时间无法同步，想要有一个时间同步功能就好了，现在已经买了zotero的同步功能，十分完美，能加上这一点就太太完美了。
文科生不清楚在技术上能否实现，感谢大大～

**Comment by volatile-static** [comment]
本来就是同步的。你是zotero6的话需要手动设置一下，具体如何操作可以去issue里找
